build_web_compilers/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Only call `window.postMessage` during initialization if the current context
   is a `Window`.
+- Added more javascript code to dev bootstrap for hot-reloading support
```

Comment:
I think this version has already been published, so we need a new `## 0.4.4-dev` heading here (and should update the pubspec as well)